<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boss Bildirim UygulamasÄ±</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Boss Timer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>402</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="karanda_btn">
       <property name="text">
        <string>Karanda</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bulgasal_btn">
       <property name="text">
        <string>Bulgasal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="kzarka_btn">
       <property name="text">
        <string>Kzarka</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="offin_btn">
       <property name="text">
        <string>Offin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="garmoth_btn">
       <property name="text">
        <string>Garmoth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="kutum_btn">
       <property name="text">
        <string>Kutum</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uturi_btn">
       <property name="text">
        <string>Uturi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nouver_btn">
       <property name="text">
        <string>Nouver</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="golden_pig_king_btn">
       <property name="text">
        <string>Golden Pig King</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sangoon_btn">
       <property name="text">
        <string>Sangoon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quint_btn">
       <property name="text">
        <string>Quint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="muraka_btn">
       <property name="text">
        <string>Muraka</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="vell_btn">
       <property name="text">
        <string>Vell</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="black_shadow_btn">
       <property name="text">
        <string>Black Shadow</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>201</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Notification Timers</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_1min">
       <property name="text">
        <string>1 min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_3min">
       <property name="text">
        <string>3 min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_5min">
       <property name="text">
        <string>5 min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_10min">
       <property name="text">
        <string>10 min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_15min">
       <property name="text">
        <string>15 min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_30min">
       <property name="text">
        <string>30 min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_60min">
       <property name="text">
        <string>60 min</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>581</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="start_btn">
       <property name="text">
        <string>Active</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_btn">
       <property name="text">
        <string>Deactive</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>581</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minimize_to_tray_btn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimize to Tray</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
